wiki/en/Software-Manual.md

```diff
@@ -84,7 +84,7 @@ The Connection Setup window shows a list of available servers together with the
 
 You can filter the list by server name or location. To list only occupied servers, enter a "#" character.
 
-If you know the IP address or URL of a private server, you can connect to it using the Server Name/Address
+If you know the IP address or URL of unregistered server, you can connect to it using the Server Name/Address
 field. An optional port number can be added after the address using a colon as a separator, e.g,
```

Comment:
> An optional port number can be added after the address using a colon as a separator

The port number is _part of_ the Server address.  It's optional only because it has a default value.